config/settings/local.py

```diff
@@ -48,3 +48,5 @@
 RUNSERVER_PLUS_PRINT_SQL_TRUNCATE = 100000
 
 DISABLE_RATELIMIT = True
+
+SMS_BACKEND = "care.utils.sms.sendSMS.CustomSMSBackendSNS"
```

Comment:
set `SMS_BACKEND = "care.utils.sms.sendSMS.CustomSMSBackendSNS"` in base settings
and for local and test use console backend